src/Components/CreateImageWizard/steps/Review/Footer/Footer.tsx

```diff
@@ -52,7 +54,9 @@ const ReviewWizardFooter = () => {
   const getBlueprintPayload = async () => {
     if (!process.env.IS_ON_PREMISE) {
       const userData = await auth?.getUser();
-      const orgId = userData?.identity?.internal?.org_id;
+      const orgId = isFedoraEnv
+        ? userData?.identity?.user?.username
+        : userData?.identity?.internal?.org_id;
```

Comment:
would be nice to abstract it away a little 🤔 I'm not sure how, but this feels slightly dirty :(

Reply:
Maybe just a helper function `getOrgId` for now? That way at least the dirty part is isolated. :thinking:

Reply:
Agreed! :)

Reply:
well, we don't really need the orgId :)  similar to the onPrem implementation.